man/rgbasm.1

```diff
@@ -291,6 +291,10 @@ This warning is enabled by
 Warn when shifting macro arguments past their limits.
 This warning is enabled by
 .Fl Wextra .
+.It Fl Wno-nested-comment
+Warn when
+.Ql /*
+appears inside a block comment.
```

Comment:
Here is what I had written in rsgbds:

```suggestion
Warn when the block comment start sequence
.Ql /*
is found inside of a block comment.
This typically indicates that the enclosing block comment is not terminated.
```

Additionally, it must be specified that the warning is enabled by default.

Reply:
> Additionally, it must be specified that the warning is enabled by default.

We say at the top:

> The following warnings are actual warning flags; with each description, the corresponding warning flag is included. Note that each of these flag also has a negation (for example, `-Wcharmap-redef` enables the warning that `-Wno-charmap-redef disables). **Only the non-default flag is listed here.**

Other enabled-y-default warnings are indicated as such by listing them as `-Wno-...`.

Reply:
Ah, then I've been misremembering. Thank you!